rebasehelper/specfile.py

```diff
@@ -584,14 +584,18 @@ def set_tag(self, tag: str, value: str, preserve_macros: bool = False) -> None:
             (?P<options>\(.+?\))?
             \s+
             (?P<value>
-                (%((?P<b>{)|(?P<s>\()))?
-                .+?
-                (?(b)})(?(s)\))
+                (
+                    %(?!{) |
```

Comment:
What about `%()` (shell expression)? Than can be multiline and can contain pretty much whatever character.

Reply:
Could you perhaps add comments, since this is a verbose regexp?

Reply:
Nice catch, forgot to consider `%(` in the lookahead and it would be consumed by `[^%]` and hence wouldn't allow multiline. Should be easy to fix. And comments are a great idea, will add some. Btw travis seems not to respond, hope it will wake up once I force push the changes...